packages/b/bibledit/package.yml

```diff
@@ -0,0 +1,26 @@
+name       : bibledit
+version    : 5.1.005
+release    : 1
+source     :
+    - https://github.com/bibledit/linux/releases/download/5.1.005/bibledit-5.1.005.tar.gz : f0b4ccd6f9406567f6c214a2a299d0bdf56480f2834459ef7b8cbd21dc332642
+homepage   : https://bibledit.org
+license    : GPL-3.0-or-later
+component  : office
+summary    : Free Bible translation software.
+description: |
+    Bibledit is the name of a professional Bible Translation Software Program. It is a collection of Bible translation tools, all conveniently in one place. Online as well as offline access is an important and handy feature. One can save the work offline, and later upload all to the Cloud when there is an active internet connection.
+builddeps  :
+    - pkgconfig(gtest)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(icu-i18n)
+    - pkgconfig(libutf8proc)
+    - pkgconfig(libxml-2.0)
+    - pkgconfig(pugixml)
+    - pkgconfig(webkit2gtk-4.0)
+    - pkgconfig(webkit2gtk-4.1)
```

Comment:
Apparently my message didn't get saved last time. Does this actually need both webkit versions? Only 4.1 appears in `abi_used_libs`.

Reply:
It built fine with only 4.1